devices/fields/tim/bdtr_common.yaml

```diff
@@ -0,0 +1,41 @@
+BDTR:
+  MOE:
+    DisabledIdle: [0, OC/OCN are disabled or forced idle depending on OSSI]
+    Enabled: [1, OC/OCN are enabled if CCxE/CCxNE are set]
+  AOE:
+    Manual: [0, MOE can be set only by software]
+    Automatic:
+      [
+        1,
+        MOE can be set by software or automatically at the next update event (if none of the break inputs BRK and BRK2 is active),
+      ]
+  "BKP,BK2P":
+    ActiveLow: [0, Break input BRKx is active low]
+    ActiveHigh: [1, Break input BRKx is active high]
+  "BKE,BK2E":
+    Disabled: [0, Break function x disabled]
+    Enabled: [1, Break function x enabled]
+  OSSR:
+    Disabled: [0, "When inactive, OC/OCN outputs are disabled"]
+    IdleLevel: [1, "When inactive, OC/OCN outputs are enabled with their inactive level"]
+  OSSI:
+    Disabled: [0, "When inactive, OC/OCN outputs are disabled"]
```

Comment:
What about something like `HiZ` here? I'm fine either way

Reply:
high impedance?

Reply:
Yes, atleast that is mentioned in RM0440 and RM0444. RM0440:

> 0:When inactive, OC/OCN outputs are disabled (the timer releases the output control which is taken over by the GPIO logic, which forces a Hi-Z state).

But I dont know what is more useful to the user